src/program/services/downloaders/alldebrid.py

```
@@ -1,10 +1,17 @@
+import time
 from datetime import datetime
 from typing import Dict, Iterator, List, Optional, Tuple
 
 from loguru import logger
 from requests import Session
 from requests.exceptions import ConnectTimeout
 
+from program.services.downloaders.models import (
+    VIDEO_EXTENSIONS,
```

Ruff (0.8.0)

`program.services.downloaders.models.VIDEO_EXTENSIONS` imported but unused

Remove unused import: `program.services.downloaders.models.VIDEO_EXTENSIONS`

(F401)